lib/winapi/thread.c

```diff
@@ -1,7 +1,64 @@
+#include <assert.h>
+#include <string.h>
 #include <processthreadsapi.h>
 #include <winbase.h>
+#include <pdclib/_PDCLIB_xbox_tls.h>
 #include <xboxkrnl/xboxkrnl.h>
 
+VOID NTAPI WinapiThreadStartup (PKSTART_ROUTINE StartRoutine, PVOID StartContext)
```

Comment:
This feels like something that should be `static`?
It's not mentioned in the header and I couldn't find docs on MSDN.

Reply:
Yeah that's a function I came up with - Windows has such a function as well, but I don't know the name they chose. Anyway, this should have been static, and it is now.